Projects/Features/SearchFeature/Sources/View/YoutubeThumbnailCell.swift

```diff
@@ -0,0 +1,36 @@
+import DesignSystem
+import SnapKit
+import Then
+import UIKit
+import Utility
+
+class YoutubeThumbnailCell: UICollectionViewCell {
```

Comment:
```suggestion
final class YoutubeThumbnailCell: UICollectionViewCell {
```
상속 대상 아니면 final 붙여주면 좋겠네요

Reply:
적용했습니다.